scripts/us_census/geojsons_low_res/download.py

```diff
@@ -90,22 +94,32 @@ def download_data(self, place='country/USA'):
         # to get the 0th element explicitly.
         assert len(geolevel[place]) == 1
         geolevel = geolevel[place][0]
+
+        level_to_fetch = geolevel
```

Comment:
Can we just use `geolevel` instead of defining another variable?

Reply:
Done.